plugin/externalPluginEntry_test.go

```diff
@@ -21,16 +23,37 @@ func (m *mockExternalPluginScript) Path() string {
 	return m.path
 }
 
-func (m *mockExternalPluginScript) InvokeAndWait(ctx context.Context, args ...string) ([]byte, error) {
-	retValues := m.Called(ctx, args)
+func (m *mockExternalPluginScript) InvokeAndWait(
```

Comment:
This refactor mirrors the calling convention in the external plugin doc, which is why I wanted to do it.